### ISSUE TITLE
A11y_api-catalog_Search_HighContrast: Focus is not visible on Search suggestion list items in High contrast black and white mode

### ISSUE THREAD

**Issue by Amulya-95** [terrajobst/apisof.net#11]
[Check out Accessibility Insights ](https://www.1eswiki.com/wiki/Accessibility_Insights)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**
#A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_api-catalog; #MAS1.4.3; #MAS2.4.7; #HighContrast; #C+E-H; #api-catalog; #Edge; #A11ySev2; #Benchmark;

**Environment Details:**
URL: https://apisof.net/

**Browser:**
Microsoft Edge 44.18362.267.0

**Repro steps:**

1)Turn on High contrast black/white mode
2) Hit the URL : "http://apisof.net" and press enter
3)Tab till "Search" field and press enter or else press "Slash" key
4)Type the content which you want to search 
5)Check whether focus is visible on Search suggestion list items in High contrast black and white mode

**Actual Result:**
Focus is not visible on Search suggestion list items in High contrast black and white mode

**Expected Result:**
Focus should be visible on Search suggestion list items in High contrast black and white mode

**User Impact:**
Keyboard only users will find it difficulty if the focus is not visible 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

[Uploading Focus not visible in high contrast black and white mode.pptx…]()

**Comment by terrajobst** [comment]
The web site no longer exists.
